ctru-rs/examples/gfx-3d-mode.rs

```diff
@@ -61,8 +62,10 @@ fn main() {
             buf.copy_from(IMAGE.as_ptr(), IMAGE.len());
         }
 
-        left.flush_buffer();
-        left.swap_buffers();
+        drop((left, right));
```

Comment:
Hmm, dropping the sides isn’t really a great thing. I understand that (especially when the double buffer swaps) the written framebuffer would be different, however, this will create some issues when more complex structs like those we could write in `citro3d-rs` take the sides as arguments.